public/styles.css

```diff
@@ -104,9 +102,9 @@ a {
 }
 
 #particles-js {
```

Comment:
The changes here fix the particles JS assets to the ends of the view width and the view height. 
Setting width and height using the view width(vw) and view height(vh) ensures the particles and images are responsive.